packages/cozy-harvest-lib/src/components/KonnectorConfiguration/KonnectorAccountTabs.jsx

```diff
@@ -91,6 +100,23 @@ const DumbKonnectorAccountTabs = props => {
         flowState={flowState}
       />
       <Divider />
+
+      {error && !isInMaintenance && (
+        <TriggerErrorInfo
+          error={error}
+          konnector={konnector}
+          action={
+            error.isSolvableViaReconnect() ? (
+              <RedirectToAccountFormButton
+                konnector={konnector}
+                trigger={initialTrigger}
+              />
+            ) : null
+          }
+          className="u-mt-1"
+        />
+      )}
+
```

Comment:
By doing this, you are changing the way we display the `TriggerErrorInfo` component, right? Before it was inside the `SwipeableViews` so I can imagine inside a container, now it's outside. 

Can you share a before / after screenshot to see the difference ?

Reply:
![before](https://user-images.githubusercontent.com/228695/195137674-28911737-f5f6-42dc-837e-a5fecf8e0058.png)

![after](https://user-images.githubusercontent.com/228695/195137738-364fbe4c-9582-479f-bfac-2e5d3c6e165c.png)

I just see that there is a difference with the button color, related to the use of the buttons component I suppose

Reply:
So we need to fix this color issue.

Reply:
I cannot reproduce this issue which came after the use of Butttons instead of Button. It looks like it was a problem related to watch and rlink I think.